docs/src/main/sphinx/admin/properties-http-client.md

```diff
@@ -17,6 +17,8 @@ The following prefixes are supported:
 
 - `oauth2-jwk` for {doc}`/security/oauth2`
 - `jwk` for {doc}`/security/jwt`
+- `exchange` for [](/admin/fault-tolerant-execution), specifically related to
+  [](fte-exchange-manager) and related to [](/admin/properties-exchange)
```

Comment:
`/admin/properties-exchange` is not about FTE. These are properties which govern behaviour of data exchanging between Trino worker nodes when FTE is NOT in use.

Reply:
so when does the http client config come into play .. fte exchange manager only ?

Reply:
No - it comes into play if FTE is disabled - in pipelined mode. When Trino workers read exchange data directly from other Trino workers. The HTTP client which is used to this worker-to-worker communication is configured with `exchange.` prefix.

Reply:
Ok .. but it also comes into play for FTE mode right?

Reply:
If so I just have to reword but have both links..

Reply:
ok .. then there was a misunderstanding at my source happening. Great to be able to clarify this. I will update the PR.

Reply:
Honestly feels a bit unfortunate that we use "exchange" for different usages

Reply:
Updated.

Reply:
>Ok .. but it also comes into play for FTE mode right?

not really :)